keras_core/backend/tensorflow/image.py

```diff
@@ -63,6 +75,130 @@ def resize(
 )
 
 
+def smart_resize(x, size, interpolation="bilinear"):
```

Comment:
We already have a backend agnostic implementation of `smart_resize` in `image_utils` (I suppose we could make it a public instead but the need has not arisen).

Reply:
@fchollet how about if we enable it in this PR by using inside `load_image`.